### PR TITLE
GroupB - Alexis

### PR DESCRIPTION
**Pull request by aakirkman10**

### CODE REVIEW

**Comment by MajesticUnic0rn** [comment]
great job

**Comment by asingh9** [comment]
@aakirkman10 your file should be named _2020-06-05-AlexisGroupB.md_. Right now it doesn't have a file ending.

File: _posts/2020-06-05-AlexisGroupB.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+title: the prompt of your choice e.g. "This summer I'm looking forward to..."
+author: Alexis Kirkman
+date: 2020-06-05
+---
+
+kayaking
+
+![kayaking](https://upload.wikimedia.org/wikipedia/commons/b/b2/Sea_Kayak.JPG)
+
+Comment your reply here.
```
Comment (tomkennon) [inline _posts/2020-06-05-AlexisGroupB.md]:
```suggestion
"Content length exceeded" when i click on your image, must be a super long kayak ride...
```

**Review by tomkennon — CHANGES_REQUESTED** [review]

File: _posts/2020-06-05-AlexisGroupB.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+title: the prompt of your choice e.g. "This summer I'm looking forward to..."
```
Comment (asingh9) [inline _posts/2020-06-05-AlexisGroupB.md]:
```suggestion
"This summer I'm looking forward to..."
```

File: _posts/2020-06-05-AlexisGroupB.md (hunk 1 above)
Comment (asingh9) [reply _posts/2020-06-05-AlexisGroupB.md]: @aakirkman10 commit tom's comment to your post

**Review by asingh9 — CHANGES_REQUESTED** [review]

File: _posts/2020-06-05-AlexisGroupB.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+---
+"This summer I'm looking forward to..."
```
Comment (asingh9) [inline _posts/2020-06-05-AlexisGroupB.md]:
```suggestion
title: "This summer I'm looking forward to..."
```

File: _posts/2020-06-05-AlexisGroupB.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+author: Alexis Kirkman
+date: 2020-06-05
+---
+
+kayaking
+
+![kayaking](https://upload.wikimedia.org/wikipedia/commons/4/4c/Boy_paddling_in_a_kayak.jpg)
+
+"Content length exceeded" when i click on your image, must be a super long kayak ride...
```
Comment (asingh9) [inline _posts/2020-06-05-AlexisGroupB.md]:
```suggestion
"Content length exceeded" when i click on your image, must be a super long kayak ride... - Tom
```

**Review by asingh9 — APPROVED** [review]

**Pull request merged**